bugbug/utils.py

```diff
@@ -19,6 +19,20 @@
 TASKCLUSTER_DEFAULT_URL = "https://taskcluster.net"
 
 
+def compress(path):
```

Comment:
Call the two functions zstd_compress and zstd_decompress, since they are specifically about zstandard.